src/test/java/bio/terra/service/filedata/DrsServiceTest.java

```diff
@@ -503,4 +660,143 @@ public void testSnapshotCache() throws Exception {
     verify(snapshotService, times(1)).retrieve(any());
     verify(snapshotService, times(1)).retrieveAvailableSnapshotProject(any());
   }
+
+  @Test
+  public void nametestMergeDrsObjects() {
```

Comment:
I think this method name may be a typo!

More meaningfully, it would be great to also see tests demonstrating / verifying behavior when 1+ required unique fields is not unique.

Reply:
derp...it totally is. I fixed it and forgot to check it in 🤦‍♂️

Reply:
Also good point re: new test.  I have it checking when there are invalid checksums but I'll add a check for that